contracts/access/manager/IAccessManager.sol

```diff
@@ -58,63 +90,302 @@ interface IAccessManager {
+     * can be increased without setback (and in the event of an accidental increase can be reset
+     * via {revokeRole}). Defaults to 5 days.
+     */
+    function minSetback() external view returns (uint32);
+
+    /**
+     * @dev Get whether the contract is closed disabling any access. Otherwise role permissions are applied.
+     */
     function isTargetClosed(address target) external view returns (bool);
 
+    /**
+     * @dev Get the role required to call a function.
+     */
     function getTargetFunctionRole(address target, bytes4 selector) external view returns (uint64);
 
+    /**
+     * @dev Get the admin delay for a target contract. Changes to contract configuration are subject to this delay.
+     */
     function getTargetAdminDelay(address target) external view returns (uint32);
 
+    /**
+     * @dev Get the id of the role that acts as an admin for the given role.
+     *
+     * The admin permission is required to grant the role, revoke the role and update the execution delay to execute
+     * an operation that is restricted to this role.
+     */
     function getRoleAdmin(uint64 roleId) external view returns (uint64);
 
+    /**
+     * @dev Get the role that acts as a guardian for a given role.
+     *
+     * The guardian permission allows canceling operations that have been scheduled under the role.
+     */
     function getRoleGuardian(uint64 roleId) external view returns (uint64);
 
+    /**
+     * @dev Get the role current grant delay.
+     *
+     * Its value may change at any point without an event emitted following a call to {setGrantDelay}.
+     * Changes to this value, including effect timepoint are notified in advance by the {RoleGrantDelayChanged} event.
+     */
     function getRoleGrantDelay(uint64 roleId) external view returns (uint32);
 
+    /**
+     * @dev Get the access details for a given account for a given role. These details include the timepoint at which
+     * membership becomes active, and the delay applied to all operation by this user that requires this permission
+     * level.
+     *
+     * Returns:
+     * [0] Timestamp at which the account membership becomes valid. 0 means role is not granted.
+     * [1] Current execution delay for the account.
+     * [2] Pending execution delay for the account.
+     * [3] Timestamp at which the pending execution delay will become active. 0 means no delay update is scheduled.
+     */
     function getAccess(uint64 roleId, address account) external view returns (uint48, uint32, uint32, uint48);
 
+    /**
+     * @dev Check if a given account currently has the permission level corresponding to a given role. Note that this
+     * permission might be associated with an execution delay. {getAccess} can provide more details.
+     */
     function hasRole(uint64 roleId, address account) external view returns (bool, uint32);
 
+    /**
+     * @dev Give a label to a role, for improved role discoverabily by UIs.
```

Comment:
```suggestion
     * @dev Give a label to a role, for improved role discoverability by UIs.
```

Reply:
The codespell check is really unreliable...

Reply:
It still gives good call outs